Makefile

```diff
@@ -12,40 +12,39 @@ SCOPE_EXPORT=scope.tar
 SCOPE_UI_BUILD_EXPORT=scope_ui_build.tar
 SCOPE_UI_BUILD_IMAGE=weave/scope-ui-build
 
-all: $(SCOPE_EXPORT)
+all: docker
+	
+docker: $(SCOPE_EXPORT)
```

Comment:
I struggle conceptually with the idea that a plain `make` target should depend on having a running Docker daemon. But I'm giving up on that.